user_data/strategies/bbrsi_scalp.py

```diff
@@ -12,6 +12,9 @@ class bbrsi_scalp(WAOStrategy):
     brain = "Freq_bbrsi_scalp"
 
     def __init__(self, config: dict):
+        self.coin = str(config.get('pairs')[0]).split('/')[0]
+        if self.coin == 'BTC' or self.coin == 'ADA':
```

Comment:
I added this condition here to help us use the separate channel and separate google sheet provided for the coins ADA and BTC with the same strategy

Reply:
Nice!